metrics.json

```diff
@@ -1,7 +1,9 @@
 [
   {"name": "TTFB", "description": "Time to First Byte", "unit": "milliseconds"},
   {"name": "render", "description": "startRender", "unit": "seconds"},
-  {"name": "firstPaint", "description": "Time to First Non-Blank Paint", "unit": "seconds"},
+  {"name": "firstPaint", "description": "Time to First Non-Blank Paint", "unit": "milliseconds"},
+  {"name": "timeToFirstMeaningfulPaint", "description": "Time to First Meaningful Paint", "unit", "milliseconds"},
+  {"name": "timeToFirstContentfulPaint", "description": "Time to First Contentful Paint", "unit", "milliseconds"),
```

Comment:
Please see the hightlighted characters, which should be comma and closing curly brace.

Reply:
Fixed.